Cython/Compiler/Builtin.py

```diff
@@ -92,13 +92,33 @@ def declare_in_type(self, self_type):
 builtin_function_table = [
     # name,        args,   return,  C API func,           py equiv = "*"
     BuiltinFunction('abs',        "d",    "d",     "fabs",
-                    is_strict_signature = True),
+                    func_type=PyrexTypes.CFuncType(
+                        PyrexTypes.c_double_type, [
+                            PyrexTypes.CFuncTypeArg(
+                                "arg", PyrexTypes.c_double_type, None)
+                        ],
+                        is_strict_signature=True, nogil=True)),
     BuiltinFunction('abs',        "f",    "f",     "fabsf",
-                    is_strict_signature = True),
+                    func_type=PyrexTypes.CFuncType(
+                        PyrexTypes.c_float_type, [
+                            PyrexTypes.CFuncTypeArg(
+                                "arg", PyrexTypes.c_float_type, None)
+                        ],
+                        is_strict_signature=True, nogil=True)),
     BuiltinFunction('abs',        "i",    "i",     "abs",
-                    is_strict_signature = True),
+                    func_type=PyrexTypes.CFuncType(
+                        PyrexTypes.c_int_type, [
+                            PyrexTypes.CFuncTypeArg(
+                                "arg", PyrexTypes.c_int_type, None)
+                        ],
+                        is_strict_signature=True, nogil=True)),
     BuiltinFunction('abs',        "l",    "l",     "labs",
-                    is_strict_signature = True),
+                    func_type=PyrexTypes.CFuncType(
+                        PyrexTypes.c_long_type, [
+                            PyrexTypes.CFuncTypeArg(
+                                "arg", PyrexTypes.c_long_type, None)
+                        ],
+                        is_strict_signature=True, nogil=True)),
     BuiltinFunction('abs',        None,    None,   "__Pyx_abs_longlong",
```

Comment:
`nogil=True` also applies here.

Reply:
It already has it; that's where I found the `CFuncType` setup originally.